pandas/tests/arithmetic/test_array_ops.py

```diff
@@ -34,3 +35,20 @@ def test_object_comparison_2d():
     right.flags.writeable = False
     result = comparison_op(left, right, operator.ne)
     tm.assert_numpy_array_equal(result, ~expected)
+
+
+@pytest.mark.parametrize(
+    "data, expected",
+    [
+        ([0, 1, 2], [0, 2, 4]),
+        ([0.0, 1.0, 2.0], [0.0, 2.0, 4.0]),
+        (["a", "b", "c"], ["aa", "bb", "cc"]),
+    ],
+)
+def test_extension_array_add(box_extension_array, data, expected):
+    # GH22606 Verify operators with Extension Array and list-likes
+    ser = Series(data)
```

Comment:
Hi @avinashpancham ,

from what I've understood of #22606, this should hold an extension array (e.g. it could hold nullable integers, like `Series([1, 2, 3], dtype='Int64')`)

Reply:
Oh I see. Updated it to test for a Series that holds an ExtensionArray.